### ISSUE TITLE
Option to view changed/reallocated land use

### ISSUE THREAD

**Issue by Butcherluke** [mostaphaRoudsari/EmergeSettlement#42]
Can an option (a button?) be added that shows what pixels have been reallocated?

**Comment by mostaphaRoudsari** [comment]
Yes! and it's a great idea. It can be as simple as adding a storke around the cells once the button is clicked.
